packages/react-mongo/types/react-mongo.d.ts

```diff
@@ -1,5 +1,10 @@
 import { Meteor } from 'meteor/meteor';
 import { Mongo } from 'meteor/mongo';
 import { DependencyList } from 'react';
-export declare const useSubscription: (factory: () => Meteor.SubscriptionHandle, deps?: DependencyList) => Meteor.SubscriptionHandle;
+declare type UseSubscriptionOptions = {
+    deps?: DependencyList;
+    updateOnReady?: boolean;
+};
+export declare const useSubscription: (factory: () => Meteor.SubscriptionHandle | void, deps?: DependencyList | UseSubscriptionOptions) => void;
```

Comment:
Is ` | UseSubscriptionOptions` already implemented? It's missing from the `useSubscriptionClient` function definition, and looking at the code it seems `updateOnReady` is always set programatically (`deps` is always passed on to `useEffect`).

Reply:
er, I think I meant to remove that. I'll review that and fix it. Thanks!

Reply:
I had created a version which would take an options object, in place of deps for setting options like updateOnRready - but I thought the `.ready()` override was a better approach, so the intent is to switch to that. What I mean is, `.ready()` turns on `updateOnReady` only when the `.ready()` method is used in the render method for a component, making everything clean and automatic. We can still provide a manual override, but I don't think it's necessary with this pattern.